src/use-cases/createUseCase.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, describe, it } from "vitest";
+import { CreateUseCase } from "./createUseCase";
+import { compare } from "bcryptjs";
+import { InMemoryUserRepository } from "../repositories/in-memory-db/inMemoryUserRepository";
+import { UserAlreadyExistsError } from "./errors/user-already-exists-error";
+
+describe('Register Use Case', () => {
+  it('should be able to register', async () => {
+    const usersRepository = new InMemoryUserRepository
+    const createUseCase = new CreateUseCase(usersRepository)
+
+    const { user } = await createUseCase.execute({
+      name: 'John',
+      surname: 'Doe',
+      email: 'johndoe@email.com',
+      password: '123456',
+    })
+
+    expect(user.id).toEqual(expect.any(String))
+  })
+
+  it('should hash user password upon registration', async () => {
+    const usersRepository = new InMemoryUserRepository
+    const createUseCase = new CreateUseCase(usersRepository)
```

Comment:
Both lines are repeating for every test. Can you use beforeEach? You can find more information about beforeEach in the documentation: https://vitest.dev/api/#beforeeach